### ISSUE TITLE
Tab navigation is uncomfortable to use

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/contacts#5050]
### Describe the bug

One example is:

### Steps to reproduce

1. Have lots of contacts

### Expected behavior

In order to reach the contact details you have to tab through all of your contact list

### Actual behavior

There should be a more comfortable way of reaching the contact details with tab navigation

This is also a general ticket for other instances of uncomfortable tab navigation

### PHP engine version

None

### Web server

None

### Database

None
